packages/pn-personafisica-login/src/utility/mixpanel.ts

```diff
@@ -14,3 +15,13 @@ export const trackEventByType = (trackEventType: TrackEventType, attributes?: ob
 
   trackEvent(trackEventType, process.env.NODE_ENV, eventParameters);
 };
+
+export function setSuperOrProfilePropertyValues<TProperty extends keyof ProfilePropertyParams>(
```

Comment:
If we decide to use only one function to manage all mixpanel tracking cases, we must
- update the `trackEventByType` in this way `trackEventByType = (trackEventType: TrackEventType, attributes?: object, propertyType: EventPropertyType = EventPropertyType.TRACK)`
- the code in profileProperties.ts must be moved to events.ts
- in the `trackEventByType` we must update the condition 
`
const eventParameters = attributes
    ? { ...events[trackEventType], ...attributes }
    : events[trackEventType];
`
because we can have empty attributes, empty events[trackEventType] or events[trackEventType] with getAttributes function that MUST NOT be passed to mixpanel

Reply:
for now we have to postpone